src/test/regress/sql/bitmap_index.sql

```diff
@@ -479,3 +479,33 @@ select * from bm_test where b = 1;
 
 -- clean up
 drop table bm_test;
+
+
+-- сheck the operation of the bitmap index on the primary and mirror after vacuum
+create table ao_table (a int) with (appendonly=true);
+insert into ao_table select generate_series(1,10);
+insert into ao_table select generate_series(1,10);
+create index bitmap_idx_ao on ao_table using bitmap(a);
+delete from ao_table where a < 8;
+
+\! psql postgres -c "select pg_sleep(5)" &
+vacuum ao_table;
+-- start_ignore
+\! gpconfig -c hot_standby -v on
+\! gpstop -afr
+-- end_ignore
+-- primary
+\c
+set enable_seqscan = off;
+select * from ao_table where a in (7, 8, 9, 10);
+-- mirror
+\c - - - 7001
+set enable_seqscan = off;
+select * from ao_table where a in (7, 8, 9, 10);
+-- clean up
+\c - - - 7000
+drop table ao_table;
+-- start_ignore
+\! gpconfig -c hot_standby -v off
```

Comment:
Maybe just reset the value to default?
```suggestion
\! gpconfig -r hot_standby
```